metaflow/plugins/cards/card_modules/basic.py

```diff
@@ -653,6 +691,18 @@ def render(self, task, components=[]):
         )
         return pt.render(RENDER_TEMPLATE, data_dict)
 
+    def render_runtime(self, task, data):
+        return self.render(task)
+
+    def refresh(self, task, data):
+        return data["components"]
+
+    def reload_content_token(self, task, data):
+        if task.finished:
+            return "final"
+        else:
+            return "runtime"
```

Comment:
TODO: Pass some information to signify layout change so that it can be handled accordingly.